.github/workflows/buildandrun.yaml

```diff
@@ -0,0 +1,36 @@
+name: buildandrun.yaml
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: install
+        run: |
+          sudo dpkg --add-architecture i386 
```

Comment:
You can create a docker image with all requirements installed and use that instead.